release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a

```diff
@@ -482,7 +482,7 @@ CONFIG_NETFILTER_ADVANCED=y
 # Core Netfilter Configuration
 #
 CONFIG_NETFILTER_NETLINK=y
-# CONFIG_NETFILTER_NETLINK_QUEUE is not set
```

Comment:
Is not set

Reply:
Yes, it was not set and required for packet manipulation from user space processes.